src/main/resources/application.properties

```diff
@@ -53,38 +53,38 @@ spring.data.mongodb.database: eiffel_intelligence
 # spring.data.mongodb.username:
 # spring.data.mongodb.password:
 # Configure the session storage expiration time for logged in users of Eiffel Intelligence front-end
-server.session-timeout: 1200
+server.session.timeout: 1200
 # Configure the database collection name to store sessions
 sessions.collection.name: sessions
 
 # Name of the database collection where aggregated objects are stored
-aggregated.collection.name: aggregated_objects
+aggregations.collection.name: aggregations
 # Configure time-to-live value (seconds) for aggregated objects. This needs to be set if
 # using the example rule set 'AllEventsRules' to avoid the storage growing too large.
 # Recommended setting is 10 minutes.
-aggregated.collection.ttlValue:
+aggregations.collection.ttl:
 # The name of the database collection where to store the mapping between
 # aggregated objects and id of the Eiffel events that contributed to those
-event_object_map.collection.name: event_object_map
+event.object.map.collection.name: event_object_map
 # Name of the database collection where subscriptions are stored
-subscription.collection.name: subscription
+subscriptions.collection.name: subscriptions
 # Name of the database collection where we store pair of 'subscription:aggregated object'
 # where notification has been sent and should not be repeated
-subscription.collection.repeatFlagHandlerName: subscription_repeat_handler
+subscriptions.repeat.handler.collection.name: subscriptions_repeat_handler
 # The waitlist contains Eiffel events which are waiting to be aggregated
-# Configure the database collection name for the waitlist
-waitlist.collection.name: wait_list
+# Configure the database collection name for the wait list
+wait.list.collection.name: wait_list
 # Set the time-to-live value for Eiffel events in the waitlist database (seconds)
-waitlist.collection.ttlValue: 600
+wait.list.collection.ttl: 600
 # Resend information for unprocessed events in wait list, time in milliseconds
-waitlist.initialDelayResend: 2000
-waitlist.fixedRateResend: 15000
+wait.list.resend.initial.delay: 2000
+wait.list.resend.fixed.rate: 15000
```

Comment:
Could be name "wait-list.resend.i..." with dash.

Reply:
I just put them back together for simplicity.

Reply:
ok